pkg/stats/stats.go

```diff
@@ -66,6 +73,30 @@ func getRebootStats(rebootTime, nodeID, pxVersion string) (map[string]string, er
 	return rebootExportable, nil
 }
 
+func GetStorkMigrationStats(mig *storkv1.Migration) (sts map[string]string, err error) {
+	migStats := make(map[string]string)
+	migStats["MigrationName"] = mig.Name
+	migStats["MigrationNamespace"] = mig.Namespace
```

Comment:
since all are constants, no need to return err here

Reply:
There is no point in naming return values.

Reply:
I changed the code at last moment and forgot to remove it, will update it.

Reply:
Updated it, Thanks. Please review.